HW06-Annotation/src/main/java/com/akproductions/hw06_main.java

```diff
@@ -0,0 +1,124 @@
+package com.akproductions;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class hw06_main {
+    public static final String TEMPLATE_NAME = "AnyTemplate";
+    public static final String TEMPLATE_TEXT = "Hi!\n %s \n With best regards, %s";
+    public static final String MESSAGE_TEXT = "How you doing?";
+    public static final String SIGN = "Vasya";
+    private MessageTemplateProvider provider;
+    private MessageBuilder messageBuilder;
+
+    public static void main(String[] args) throws Exception {
+        //входной параметр - имя класса
+        String ClassToTest = "com.akproductions.MessageBuilderImpl";
```

Comment:
```suggestion
        String classToTest = "com.akproductions.MessageBuilderImpl";
```

Переменные принято писать в camelCase